feeds/wlan-ap/opensync/patches/17-dpp-sta-events.patch

```diff
@@ -0,0 +1,1489 @@
+Index: opensync-2.0.5.0/src/lib/datapipeline/inc/dpp_events.h
+===================================================================
+--- /dev/null
++++ opensync-2.0.5.0/src/lib/datapipeline/inc/dpp_events.h
+@@ -0,0 +1,364 @@
```

Comment:
new files should go into the src folder and not the patches folder

Reply:
I agree with you on that. However, if you look at the **13-synthetic_client** patch you'll see that the **"dpp_network_probe.h"** was added in the same manner.  Wouldn't it be better if we all followed the same guidelines? In which **"src"** directory should the newly added files reside?

Reply:
I can fix this now if that's ok with the author. @AmmadRehmatCUS

Reply:
Fixed if 4041313